.github/actions/init_build.sh

```diff
@@ -7,6 +7,3 @@ set -e -u -x -o pipefail
 # (see https://github.com/laminas/laminas-diactoros/issues/117#issuecomment-1267306142).
 # The `+` suffix on `php+` indicates that we ignore the upper bound of our dependencies, see https://getcomposer.org/doc/03-cli.md#install-i
 bin/console dependencies install --composer-options="--ignore-platform-req=php+ --prefer-dist --no-progress"
-
-# Compile translation files
-php bin/console locales:compile
```

Comment:
This is no longer needed as locales are compiled by the `dependencies install` command since #17784.